verify_pr_lables.py

```diff
@@ -238,9 +238,10 @@ def get_env_var(env_var_name, echo_value=False):
     if review_missing_label:
         print('The last review already requested changes')
     else:
+        formatted_labels = "\n".join(map(lambda label: f"• {label}", valid_labels))
```

Comment:
Thank you for the suggestion @JakeCooper. Although I don't think the code as-is will have the intended result. The message looks like this:
<img width="333" alt="message_1" src="https://github.com/jesusvasquez333/verify-pr-label-action/assets/3729513/674855c7-1162-43cf-8f25-1ee070e8377f">

Instead, I think you want the code to be:
```
formatted_labels = "\n".join(f"* `{label}`" for label in valid_labels)
pr.create_review(
    body='This pull request does not contain a valid label.\n\nPlease '
         f'add one of the following labels:\n{formatted_labels}',
    event='REQUEST_CHANGES')
```
(also, the `map` and `lambda` are not really needed, the simplified code above will produce the same result).

With this change the message looks like this (which I think if what you wanted?):
<img width="347" alt="message_2" src="https://github.com/jesusvasquez333/verify-pr-label-action/assets/3729513/a92a8f07-1c4d-40aa-a8e6-5ddd8ad86f8c">